src/graph_compute/bsp_scheduling.h

```diff
@@ -1,7 +1,7 @@
 #ifndef BSP_SCHEDULING_H_
 #define BSP_SCHEDULING_H_
 
-#if D0_BSP
+#if D0_BSP || BASELINE
```

Comment:
I understand, but this is more than just a naming change -- BASELINE is a run configuration that the scripts depend on and use. If we want to compare against D0_BSP, we can just use the D0_BSP flag. If you make BASELINE mean the same thing as D0_BSP, then there will be no way to run what used to be the BASELINE configuration.